static/install.sh

```diff
@@ -55,6 +55,12 @@ fi
 
 echo "Downloading installation package..."
 curl -s $PACKAGE -o /tmp/opta.zip
+ls /tmp/opta.zip
+if [[ $? != 0 ]]
+  echo "Version $VERSION not found."
+  echo "Please check the Opta Version if you have defined any. It should be of the format: x.y.z"
```

Comment:
Just say, "Please check the available versions at https://github.com/run-x/opta/releases"

Reply:
> Just say, "Please check the available versions at https://github.com/run-x/opta/releases"

Sure